deltachat-ios/Controller/NewContactController.swift

```diff
@@ -108,6 +107,26 @@ class NewContactController: UITableViewController {
         super.didReceiveMemoryWarning()
         // Dispose of any resources that can be recreated.
     }
+
+
+    // MARK: - coordinator
```

Comment:
we could move stuff from the coordinators to such a `// MARK: -` - this way, we can easily create coordinators as needed - if really needed.